src/cose/test_cose.rs

```diff
@@ -206,3 +206,63 @@ fn test_cose_sign_verify_tampered_signature() {
     assert!(verify_result.is_err());
     assert_eq!(verify_result, Err(CoseError::VerificationFailed));
 }
+
+#[test]
+fn test_cose_sign_verify_rsa() {
```

Comment:
I feel like these all the tests in this file duplicate a lot of code. We should probably make them share some of the code.

Reply:
Ok - I filed #36 for this.